README.md

```diff
@@ -1,59 +1,78 @@
+
+Diagnostics highlight potential problems in your configuration, such as invlaid values. 
+
 * Diagnostics on Liberty properties and variables
 
-![Screenshot showing diagnostics marking an invalid value for a Liberty property in a bootstrap.properties file. Hovering over the diagnostic will provide more details.](./docs/images/property-diagnostic.png "Diagnostics on Liberty properties in bootstrap.properties")
-![Screenshot showing diagnostics marking an invalid value for a Liberty variable in a server.env file. Hovering over the diagnostic will provide more details.](./docs/images/variable-diagnostic.png "Diagnostics on Liberty variables in server.env")
+![Screen capture showing diagnostics marking an invalid value for a Liberty property in a bootstrap.properties file. Hovering over the diagnostic will provide more details.](./docs/images/property-diagnostic.png "Diagnostics on Liberty properties in bootstrap.properties")
+![Screen capture showing diagnostics marking an invalid value for a Liberty variable in a server.env file. Hovering over the diagnostic will provide more details.](./docs/images/variable-diagnostic.png "Diagnostics on Liberty variables in server.env")
+
 * Diagnostics for Liberty XML configs
 
-![Screenshot showing diagnostics marking an invalid feature defined in a server.xml file. Hovering over the diagnostic will provide more details.](./docs/images/feature-diagnostic.png "Diagnostics on Liberty features in server.xml")
+![Screen capture showing diagnostics marking an invalid feature defined in a server.xml file. Hovering over the diagnostic will provide more details.](./docs/images/feature-diagnostic.png "Diagnostics on Liberty features in server.xml")
 
 ### Liberty dev mode schema generation
-If [Liberty Maven Plugin](https://github.com/OpenLiberty/ci.maven) or [Liberty Gradle Plugin](https://github.com/OpenLiberty/ci.gradle) is configured with the Liberty project, Liberty Config Language Server will automatically generate a schema file based on the Liberty runtime and version to provide relevant information about the supported `server.xml` elements and Liberty features. A minimum version of Liberty Maven Plugin 3.7.1 or Liberty Gradle Plugin 3.5.1 is recommended. 
+If the [Liberty Maven Plugin](https://github.com/OpenLiberty/ci.maven) or [Liberty Gradle Plugin](https://github.com/OpenLiberty/ci.gradle) is configured with the Liberty project, Liberty Config Language Server automatically generates a schema file based on the Liberty runtime and version to provide relevant information about the supported `server.xml` elements and Liberty features. A minimum version of the Liberty Maven Plugin 3.7.1 or Liberty Gradle Plugin 3.5.1 is recommended.  If an earlier version of either plugin is used, the schema file will not be regenerated when additional features are installed. This limitation might result in lack of hover, completion, or diagnostic support for elements related to those newly installed features.
```

Comment:
it's not clear how this section fits in with the previous information- is this discussing a separate set of features from those already mentioned? Or do all features depend on this schema generation?

I can't tell if this is just to identify recommended minimum levels for all features, or just some subset that pertains to dev mode and maven

Reply:
All the support for `server.xml` (and other xml files that are Liberty server configuration as defined earlier) depend on the schema generation. If the schema cannot be generated, then a default one is used (which currently matches 22.0.0.9 - wonder if we should mention that?).

Reply:
ok- I'm going to revise my suggestion